spring-cloud-task-batch/src/test/java/org/springframework/cloud/task/batch/partition/SimpleCommandLineArgsProviderTests.java

```diff
@@ -30,6 +30,7 @@
  */
 public class SimpleCommandLineArgsProviderTests {
 
+	//TODO remove when SimpleCommandLineArgsProvider is removed.
```

Comment:
Why would `SimpleCommandLineArgsProvider` be removed?  Do you mean the constructor?